packages/browser/src/browser/index.ts

```diff
@@ -180,7 +180,6 @@ async function registerPlugins(
   writeKey: string,
   legacySettings: LegacySettings,
   analytics: Analytics,
-  opts: InitOptions,
   options: InitOptions,
```

Comment:
Woah, this is so confusing, why was it this way?

Reply:
@oscb This has been confusing me for ... checks wash... since the very first time I saw this codebase almost 2 years ago. I feel like we should break out the champagne once this gets merged 😄